src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/UnnamedSymbolCompletionProvider.cs

```diff
@@ -149,12 +149,10 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
             }
         }
 
-        internal override Task<CompletionChange> GetChangeAsync(
```

Comment:
`AbstractImportCompletionProvider` is also overriding the internal overload. Is that a problem?
If it should never be overridden, could we make it non-virtual, or actually delete it?

Reply:
Kinda. Omnisharp uses private reflection to resolve import completion at a later point (during the resolve stage, when the added overhead of reflection isn't a burden), and the Roslyn lsp does not have import completion enabled at all. However, the import provider actually uses the extra args from the internal GetChangeAsync (iirc the internal one was added _for_ import completion to receive those extra args), and this provider does not.